src/main/webapp/app/shared/sidebar/sidebar.component.scss

```diff
@@ -41,3 +41,65 @@
         display: none !important;
     }
 }
+
+.dropdown-container,
+.btn-primary,
+.dropdown-menu,
+.dropdown-item {
+    box-sizing: border-box;
+}
+
+.dropdown-container {
+    position: relative;
+    width: 100%;
+    display: block;
```

Comment:
you can also do this with the bootstrap class attributes directly in the html template